CatOrganization/GoTtBGtCG
# [I-3] Adding assets to unity project

Validation steps: 

Open unity and you should see something like this.

Comment if it doesnt work, approve if you do see that
